.github/workflows/main.yml

```diff
@@ -130,5 +130,54 @@ jobs:
         with:
          name: app reports
          path: app/build/test-results
-      
-      
+
+  bazel_tests:
+    name: Bazel tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        id: cache
+        with:
+          path: ~/oppia-bazel/bazel-bin/src/bazel
+          key: ${{ runner.os }}-bazel-${{ hashFiles('**/*.bazel') }}}
+      - name: Setup bazel
```

Comment:
Nit: ``Bazel`` here & elsewhere when referring to it in user-readable text.

Reply:
okay